rdmd.d

```diff
@@ -59,6 +59,15 @@ version(unittest) {} else
 int main(string[] args)
 {
     //writeln("Invoked with: ", args);
+    string binPath = thisExePath();
+    if (binPath.exists && binPath.isFile && filenameCmp(baseName(binPath),
+                                                        baseName(args[0])) == 0)
```

Comment:
Ah, ok, you didn't say what the problem was, so I thought you were referencing something like the wrapper script you mentioned below.  I guess this means the last check will fail on Windows, and it'll fall back to the dmd from `PATH` instead.  I'll just remove this check, doesn't do much even when it works.